src/styles/blocks/circle.scss

```diff
@@ -0,0 +1,66 @@
+.container__circle {
+  @include circle-size;
+  border-radius: 50%;
+
+  &--color--1 {
+    @include circle-element-position;
+    background-color: $main-color;
+    border: 30px solid $first-color;
+  }
+
+  &--color--2 {
+    background-color: $main-color;
+    border: 60px solid $second-color;
+  }
+
+  &--color--3 {
+    background-color: $third-color;
+  }
+
+  &--color--4 {
+    background-color: $main-color;
+    border: 20px solid $fourth-color;
+
+  }
+
+  &--color--5 {
+    background-color: $main-color;
+    border: 20px solid $third-color;
+  }
+
+  &--color--6 {
+    background-color: $second-color;
+  }
+
+  &--color--7 {
+    @include circle-element-position;
+    background-color: $main-color;
+    border: 10px solid $fourth-color;
+  }
+
+  &--color--8 {
+    background-color: $main-color;
+    border: 60px solid $first-color;
+  }
+
+  &--color--9 {
+    background-color: $main-color;
+    border: 60px solid $fourth-color;
+  }
+
+  &--color--10 {
+    @include circle-element-position;
+    background-color: $main-color;
+    border: 20px solid $third-color;
+  }
+
+  &--color--11 {
+    background-color: $main-color;
+    border: 30px solid $first-color;
+  }
+
+  &--color--12 {
+    background-color: $second-color;
+  }
```

Comment:
You should probably create a variable for each border with the values, they are repeated several times:
 `10px solid ,20px solid ,30px solid ,60px solid `

Reply:
or make @mixins with arguments but no special needs of this in such task!)